eth-brownie/brownie
# feat: replaces usages of deprecated eth_abi package methods

### What I did
Running tests using brownie outputs lots of `eth-abi` package deprecation warnings like:
```
DeprecationWarning: abi.encode_abi() and abi.encode_abi_packed() are deprecated and will be removed in version 4.0.0 in favor of abi.encode() and abi.encode_packed(), respectively.
```
Source: https://github.com/ethereum/eth-abi/blob/v2.2.0/eth_abi/abi.py#L11
This PR addresses fixes the warnings.

### How I did it
Replaces deprecated methods:

- `encode_abi` -> `encode`
- `decode_abi` -> `decode`

### How to verify it
Run the tests using brownie.network.contract functionality and notice lack of above warnings.

### Checklist
